support-frontend/assets/pages/paper-subscription-landing/components/paperPrices.jsx

```diff
@@ -90,12 +90,11 @@ const getPlans = (
       title: getTitle(productOption),
       price: showPrice(price),
       href: paperCheckoutUrl(fulfilmentOption, productOption, promoCode),
-      onClick: sendTrackingEventsOnClick(
-        'subscribe_now_cta',
-        'Paper',
-        null,
-        [productOption, fulfilmentOption].join(),
-      ),
+      onClick: sendTrackingEventsOnClick({
+        id: `subscribe_now_cta-${[productOption, fulfilmentOption].join()}`,
+        product: 'Paper',
+        componentType: 'ACQUISITIONS_BUTTON',
+      }),
```

Comment:
Updated the `id` to include `productOption` and `fulfilmentOption` so we can get more info on the CTA selected. Corresponding Trello Card [here](https://trello.com/c/ynLFxBjM/3567-update-click-tracking-component-id-on-pricing-cards-on-the-digital-subs-paper-product-landing-pages-to-illustrate-period).